### ISSUE TITLE
Animating Dynamic Fields

### ISSUE THREAD

**Issue by chathuraa** [rekit/antd-form-builder#110]
Hi there,

Is it possible to animate the dynamic fields?

Thanks,
Chat.

**Comment by berakoc** [comment]
Yes. If you use an animation library like [Framer Motion](https://www.framer.com/motion/) you can manipulate class names or add a wrapper and create animations of your choice.
